usagov/vote-gov-drupal
# VOTE-2903 incorporate latest data structure changes from nvrf

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2903

## Description

Update NVRF to latest v0.18.0

## Deployment and testing

### Post-deploy steps

1. run `npm i` and `npm run build` in vote_nvrf module directory
2. `lando drush cr`

### QA/Testing instructions

1. visit the NVRF page in English and Spanish and confirm that you are able to complete the form to completion.

## Checklist for the Developer




## Checklist for the Peer Reviewers
